### PR TITLE
Add branch protection aliases

### PR DESCRIPTION
**Pull request by Rayman**

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Documentation missing

**Comment by Rayman** [comment]
To test, please update first:
```bash
$ tue-get update robot-user
```

File: installer/targets/robot-user/setup [hunk 1]
```diff
@@ -115,3 +115,56 @@ PS1='[\[\033[01;34m\]\w\[\033[00m\]]$(__git_ps1)\n'\
 '$(ROS_MASTER_NAME)${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\] \$ '
 
 alias scores='scp amigo@192.168.2.110:/home/amigo/database/per_challenge/scores.html /tmp/scores.html && google-chrome /tmp/scores.html'
+
+function _github-prompt() {
+	if [[ -z $myuser || -z $mypass ]]
+	then
+		read  -p "Enter Username: " myuser
+		read -s -p "Enter Password: " mypass
```
Comment (Timple) [inline installer/targets/robot-user/setup]: Password variable is never cleared. Would this be usefull or desired?

**Comment by MatthijsBurgh** [comment]
Is it possible to create a function which shows all open pull requests for all repos? Maybe with the option to only show when you are assigned or marked as reviewer. Both separate options.

File: installer/targets/robot-user/setup (hunk 1 above)
Comment (LoyVanBeek) [inline installer/targets/robot-user/setup]: Yes, unsetting this variable is probably a good idea: 
```bash
unset mypass
```

Comment (MatthijsBurgh) [inline installer/targets/robot-user/setup]: Can this be done with a delay?

Comment (LoyVanBeek) [inline installer/targets/robot-user/setup]: Yes, it is possible, but perhaps its better to re-evaluate whether this is really the way to go. 
GitHub will prompt for password when needed and with credential-helper set up properly, this won't be needed I think.

**Comment by Rayman** [comment]
You can find all the PRs here: https://github.com/pulls?user=tue-robotics
All the issues: https://github.com/issues?user=tue-robotics

File: installer/targets/robot-user/setup [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
-{                                                                                              
-    if [ -z "$2" ]                                                                             
-    then                                                                                       
-        echo -e "\e[0;31mPlease provide <robot-name> <robot-env>\033[0m"                                     
-        return 1                                                                               
+{
+    if [ -z "$2" ]
+    then
+        echo -e "\e[0;31mPlease provide <robot-name> <robot-env>\033[0m"
+        return 1
```
Comment (reinzor) [inline installer/targets/robot-user/setup]: What does this have to to with this PR?

**Comment by MatthijsBurgh** [comment]
Please fix this tomorrow

**Comment by MatthijsBurgh** [comment]
@Rayman, are we using this or not? otherwise close this PR

File: installer/targets/robot-user/setup (hunk 2 above)
Comment (Rayman) [inline installer/targets/robot-user/setup]: My Sublime configuration removes spaces at EOL

**Comment by Rayman** [comment]
I don't know why you guys are so strict around this PR. Its just some small utility functions which I have already used.

**Comment by MatthijsBurgh** [comment]
I want to keep it clean. So please remove the credential stuff and merge or close this PR. Is that ok?

**Pull request closed without merging**